Changes

```diff
@@ -387,6 +387,9 @@ Working version
 - #9824, #9837: Honour the CFLAGS and CPPFLAGS variables.
   (Sébastien Hinderer, review by David Allsopp)
 
+- #9895, #9523: No longer install VERSION file
```

Comment:
```suggestion
- #9895, #9523: Avoid conflict with C++20 by not installing VERSION to the OCaml
  Standard Library directory.
```
(sorry for such a small tweak, but it seems worth being clear _why_ the change is being made as well as _what_ it is!)

Reply:
No problem, I commited your suggested change.